teachers_digital_platform/jinja2/teachers_digital_platform/prototypes/crt-survey.html

```diff
@@ -1,148 +1,339 @@
+            <li>Skim the lesson plans.</li>
+            <li>Select <strong><em>yes</em></strong> for the components that are addressed, and <strong><em>no</em></strong> for those that are not.</li>
+        </ul>
+        <div class="m-notification
+                    m-notification__visible
+                    u-mb30
+                    u-mt30
+                    u-mw670">
+            <div class="m-notification_content">
+                <div class="m-notification_message">
+                    <div class="m-notification_message">
+                        <h4>This dimension has <em>essential</em> and <em>beneficial</em> components.</h4>
+                        <p><strong><em>Essential components</em></strong> have been shown to positively impact student learning.<br><strong><em>Beneficial components</em></strong> hold promise for positive impact on student learning, but may be more relevant and useful for some reviewers. Beneficial components are marked with a note; all others are essential. You must answer all components.</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <hr class="a-hr
+                    a-hr__gray40
+                    u-mb30
+                    u-mt30">
+    </div>
+    <div class="block block__flush-top">
+        <h3 class="h2">{# {% include icons/approved-round.svg %} #}Criterion 1: Completed criterion <span class="u-fc-gray">(complete)</span></h3>
+        <div class="lead-paragraph
+                    u-mb45">
+            <p>Does the curriculum address grade-level appropriate topics for saving and investing?</p>
+        </div>
+        <div class="o-survey">
+            <div class="o-survey_number">
+                <div class="h3">1.1</div>
+            </div>
+            <div class="o-survey_indicator">
+                <div class="h3">Indicator</div>
                 <p>People save for the future, and might have different goals for saving and make different choices about how to save.</p>
-                <h4>Component</h4>
-                <p>One can save money in various ways, including at home in a piggy bank or at a commercial bank, credit union, or savings and loan institution.</p>
-                <p>Saving helps individuals act on future opportunities, meet short-term and long-term goals, and address financial emergencies.</p>
-                <div class="m-form-field m-form-field__radio">
-                    <input class="a-radio" type="radio" name="radio_1" id="test_1">
-                    <label class="a-label" for="test_1">Yes</label>
+            </div>
+            <div class="o-survey_components">
+                <div class="h3">Component</div>
+                <div class="o-survey_component">
+                    <div class="o-survey_question">
+                        <p>One can save money in various ways, incl home in a piggy bank or at a commercial union, or savings and loan institution.</p>
```

Comment:
Lost the "uding" in "including" in the move from the previous code to this code :)

Reply:
Fixed